tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -1294,3 +1305,113 @@ async def test_notification_rpcs(wallet_rpc_environment: WalletRpcTestEnvironmen
     notification = (await client_2.get_notifications())[0]
     assert await client_2.delete_notifications([notification.coin_id])
     assert [] == (await client_2.get_notifications([notification.coin_id]))
+
+
+@pytest.mark.parametrize(
+    "rpc_request, rpc_response",
+    [
+        # Valid signatures
```

Comment:
Recreating these would require access to a particular set of mnemonics I would think?  I'm not sure if it is better to have a public set for stuff like this or if we just accept that any modification will often be a complete change including the address and mnemonics used.

Reply:
I'm mostly in the camp of tests should run against static data, so that was my reasoning for baking in pre-computed signature data here.

As you mention, recreating this exact set of test data would require using the same mnemonic, but my expectation is any updates to this would simply create new test data using a new throw-away mnemonic. I could add a comment explaining that in more detail if you think that would be helpful. I included the CLI for generating the test data, but maybe that's not clear on its own.

Reply:
Not a huge deal, but a comment might save someone some time looking around to see if there's an existing key to use.

Reply:
Added